assets/js/components/KeyMetrics/MetricTileTable.js

```diff
@@ -90,9 +97,9 @@ export default function MetricTileTable( props ) {
 						/>
 					) }
 					{ ! loading && (
-						<table className="googlesitekit-km-widget-tile__table">
-							<tbody>{ tbody }</tbody>
-						</table>
+						<div className="googlesitekit-km-widget-tile__table">
+							{ tbody }
```

Comment:
As the markup is no longer using a `table`, this `tbody` name seems a little out of place. I'd suggest renaming it, how about `tileBody`?

```suggestion
							{ tileBody }
```

Reply:
Updated.